### DISCUSSION TITLE
[Support]: Is there a way to get event preview clips via HA Notification API?

### DISCUSSION THREAD

**Discussion by aeozyalcin** [blakeblackshear/frigate#15750] [General Support]
### Describe the problem you are having

I am looking for a way to retrieve the low-quality/sped up preview clips (either mp4 or gif) for any given event using the Notification API mentioned [here](https://deploy-preview-13787--frigate-docs.netlify.app/integrations/home-assistant/#notification-api). Notification API only gives access to snapshot, clip and thumbnail. It would be awesome for it to also support the preview.

I was able to use the HTTP API to get the preview, but this won't work for sending to user phones as notifications, as my Frigate instance is not exposed to the internet, and requires HA ingress. Even with the host phone logged into HA, notifications that reference assets that require auth don't seem to work. Looking forward to some guidance on this. Thanks! 

### Version

v15 beta 4

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
NA
```

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

Reolink

**Comment by NickM-27** [comment]
The notification API gives access to much more than that, the documentation just hasn't been updated

https://github.com/blakeblackshear/frigate-hass-integration/blob/2e5e6903fdea0ede296ff6882baf3e7d0f5fccca/custom_components/frigate/views.py#L242-L256
